docs/general/ad-filtering/create-own-filters.md

```diff
@@ -3437,22 +3437,22 @@ CSS rules may operate differently [depending on the platform](#cosmetic-rules-pr
 
 :::info Adblock Plus compatibility
 
-In AdGuard products that use **CoreLibs version 1.18 or later**, you can also use element hiding rules to inject CSS.
-For example:
+In AdGuard products that use **CoreLibs version 1.18 or later**, you can also use element hiding rules to inject
+a `remove: true` declaration:
 
 ```adblock
-example.org##body { background-color: green; }
-abptestpages.org##.testcase-inline-css-ehe {background-color: #c70d2c;}
+example.org##body { remove: true; }
 ```
 
 This usage is discouraged in favor of using [CSS rules](#cosmetic-css-rules) and is only supported for compatibility with filter lists written for Adblock Plus.
 
-Element hiding exceptions (`#@#`) can match such a rule both by its selector part and by the full rule text.
-For example, the rule `example.org##body { background-color: green; }` can be disabled by any of the following exception rules:
+Element hiding exceptions (`#@#`) are matched by the selector part only, ignoring the declarations block part.
```

Comment:
```suggestion
Element hiding exceptions (`#@#`) are matched by the selector part only, ignoring the declaration part.
```

Reply:
`{ remove: true; }` is the declarations block, I'm refering to the part of the rule that is the declarations block :)

Reply:
I see
ignore my suggestion then :)